charts/kube-prometheus-stack/templates/exporters/kubelet/servicemonitor.yaml

```diff
@@ -69,7 +71,11 @@ spec:
     scrapeTimeout: {{ .Values.kubelet.serviceMonitor.scrapeTimeout }}
     {{- end }}
     honorLabels: {{ .Values.kubelet.serviceMonitor.honorLabels }}
+    {{ if .Values.kubelet.serviceMonitor.trackTimestampsStaleness }}
+    honorTimestamps: true
+    {{- else }}
     honorTimestamps: {{ .Values.kubelet.serviceMonitor.honorTimestamps }}
+    {{- end }}
     trackTimestampsStaleness: {{ .Values.kubelet.serviceMonitor.trackTimestampsStaleness }}
```

Comment:
Could add the link to the very detailed comment (https://github.com/prometheus-community/helm-charts/pull/5063#issuecomment-2545374849) to the corresponding trackTimestampsStaleness property in the values.yaml?